services/src/main/java/org/jboss/windup/web/services/service/AnalysisContextService.java

```diff
@@ -6,10 +6,8 @@
 import javax.ws.rs.BadRequestException;
 import javax.ws.rs.NotFoundException;
 
-import org.jboss.windup.web.services.model.AnalysisContext;
-import org.jboss.windup.web.services.model.MigrationProject;
+import org.jboss.windup.web.services.model.*;
```

Comment:
do you have imported eclipse formatter from https://github.com/windup/windup/blob/master/ide-config/Eclipse_Code_Format_Profile.xml or is this wildcarding of imports intentionally?